integration-tests/test/native-eth-ovm-calls.spec.ts

```diff
@@ -24,47 +24,68 @@ describe('Native ETH value integration tests', () => {
-        await wallet.provider.getBalance(other.address),
-      ]
-    }
-
-    const checkBalances = async (
-      expectedBalances: BigNumber[]
-    ): Promise<void> => {
-      const realBalances = await getBalances()
-      expect(realBalances[0]).to.deep.eq(expectedBalances[0])
-      expect(realBalances[1]).to.deep.eq(expectedBalances[1])
-    }
+  describe('Basic behavior', () => {
+    it('should allow an L2 EOA to send to a new account and back again', async () => {
+      const getBalances = async (): Promise<BigNumber[]> => {
+        return [
+          await wallet.provider.getBalance(wallet.address),
+          await wallet.provider.getBalance(other.address),
+        ]
+      }
+
+      const checkBalances = async (
+        expectedBalances: BigNumber[]
+      ): Promise<void> => {
+        const realBalances = await getBalances()
+        expect(realBalances[0]).to.deep.eq(expectedBalances[0])
+        expect(realBalances[1]).to.deep.eq(expectedBalances[1])
+      }
+
+      const value = 10
+      await fundUser(env.watcher, env.l1Bridge, value, wallet.address)
+
+      const initialBalances = await getBalances()
+
+      const there = await wallet.sendTransaction({
+        to: other.address,
+        value,
+        gasPrice: 0,
+      })
+      await there.wait()
 
-    const value = 10
-    await fundUser(env.watcher, env.l1Bridge, value, wallet.address)
+      await checkBalances([
+        initialBalances[0].sub(value),
+        initialBalances[1].add(value),
+      ])
 
-    const initialBalances = await getBalances()
+      const backAgain = await other.sendTransaction({
+        to: wallet.address,
+        value,
+        gasPrice: 0,
+      })
+      await backAgain.wait()
 
-    const there = await wallet.sendTransaction({
-      to: other.address,
-      value,
-      gasPrice: 0,
+      await checkBalances(initialBalances)
     })
-    await there.wait()
 
-    await checkBalances([
-      initialBalances[0].sub(value),
-      initialBalances[1].add(value),
-    ])
+    it('should allow ETH to be burned by sending to zero address', async () => {
+      const Factory__EthBurner: ContractFactory = await ethers.getContractFactory(
+        'EthBurner',
+        wallet
+      )
+      const EthBurner: Contract = await Factory__EthBurner.deploy()
+      await EthBurner.deployTransaction.wait()
 
-    const backAgain = await other.sendTransaction({
-      to: wallet.address,
-      value,
-      gasPrice: 0,
-    })
-    await backAgain.wait()
+      const amount = 10
+      await fundUser(env.watcher, env.l1Bridge, amount, EthBurner.address)
+      const initialBalance = await wallet.provider.getBalance(EthBurner.address)
 
-    await checkBalances(initialBalances)
+      const tx = await EthBurner.burnETH(amount)
+      await tx.wait()
+
+      const postBalance = await wallet.provider.getBalance(EthBurner.address)
+      expect(initialBalance.sub(postBalance)).to.eq(amount)
+    })
```

Comment:
What do you think about sending a tx via RPC to the zero address? this also tests that they aren't blocked at the RPC level. That is also a good test to have for the verifier/replica, ie the `to` is set to `address(0)` and that is synced correctly

Reply:
That's what I originally tried, but I ran into [this](https://github.com/ethereum-optimism/optimism/blob/aba77c080d1bb951cab2084e6208c249e33aaef8/l2geth/eth/api_backend.go#L313).

Reply:
I believe that should be able to be removed now, its only in there due to `address(0)` meaning create previously